src/aliases.js

```diff
@@ -0,0 +1,32 @@
+import { promises as fs } from 'fs'
+import { version as processVersion } from 'process'
+
+import findUp from 'find-up'
+
+export const NODE_VERSION_ALIAS = '.'
+export const NODE_VERSION_FILES = ['.node-version', '.nvmrc', '.naverc']
+
+export const CURRENT_NODE_ALIAS = '_'
+
+export const resolveNodeVersionAlias = async ({ cwd } = {}) => {
```

Comment:
It would be nice to add a test for the case where `cwd` is not specified, i.e. the default value is used.

One way to do it could be to something like this:

```js
import { cwd as getCwd, chdir } from 'process'

// We are changing the global `cwd` so this must be serial
test.serial('Option cwd defaults to the current directory', async (t) => {
  const currentCwd = getCwd()
  chdir(`${__dirname}/fixtures/nvmrc-project`)
  try {
    const versionRange = await normalizeNodeVersion('.')
    t.is(versionRange, 'v12.14')
  } finally {
    chdir(currentCwd)
  }
})
```